bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/model/User.kt

```diff
@@ -0,0 +1,10 @@
+package io.bucketeer.sdk.android.internal.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class User(
+  val id: String,
+  val data: Map<String, String> = emptyMap(),
```

Comment:
Just to let you know while the swagger file is updating.
The user `data` is the only data that is not required in all requests. The server only checks the user ID.
Anything else is non-null, as you implemented.

Also, the `tagged_data` is used only on the server side, which can be empty as you implemented.

Reply:
Thanks!
Then can we remove `tagged_data` from `User`?

Reply:
Yes, please remove it.

Reply:
removed: [`9a38bd4` (#58)](https://github.com/ca-dp/bucketeer-android-sdk/pull/58/commits/9a38bd488f889e8d6e37efb7cbf7b7edc4fadd9b)